src/test/components/SignupPage.test.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import SignupPage from '../../views/SignupPage';
+
+const props = [{
+  message: {message: "Email is required"},
+  close: () => {
+    return;
+  }
+}];
+
+describe('<SignupPage Test Suite>', () => {
```

Comment:
#### Nitpick
- Is this all you have for the test regarding the signup page?

Reply:
Alright I have added more test cases